protected/views/patient/crud/_form.php

```diff
@@ -297,7 +297,14 @@
         <?php echo CHtml::hiddenField('Patient[gp_id]', $patient->gp_id, array('class'=>'hidden_id')); ?>
     </div>
     <div id="no_gp_result" class="row field-row hide">
-        <div class="large-offset-4 large-8 column selected_gp end">No result</div>
+        <div class="large-offset-4 large-8 column selected_gp end">No result
```

Comment:
Maybe wrap the text here in `<p>` tags? That might improve the on-screen formatting a bit.